journey-map/public/index.mjs

```diff
@@ -101,18 +101,16 @@ const switchJourney = async (targetJourney, targetState) => {
 };
 
 const setupHeader = () => {
```

Comment:
I wonder if we should still include some of the 'frequent' options in the header - i.e. `NEW_P2_IDENTITY`? I'd quite like to avoid extra clicks where possible...

Do you think it'd also be worth making the `h1` (IPV Core Journey Map) a link back to the homepage?